README.md

```diff
@@ -23,6 +23,27 @@ used to set Spack up on the desktops:
     $ . spack/share/spack/setup-env.sh
     $ spack compiler find
 
+For Ubuntu 20.04:
+
+    $ git clone https://github.com/BlueBrain/spack.git
+    $ mkdir ~/.spack
+    $ cp spack/sysconfig/ubuntu-20.04/*.yaml ~/.spack
+    $ sed -e 's/#.*//g' spack/sysconfig/ubuntu-20.04/packages|xargs -r sudo apt-get install --assume-yes
+    $ . spack/share/spack/setup-env.sh
+    $ spack compiler find
+
+Since Ubuntu 20.04 dropped python2 support, we need to set python3 as the default python:
+
+    $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
```

Comment:
In `packages.yaml`, you list 3.8.2, though…?

Reply:
it is because there were 3 numbers before. However, I do not have a python3.8.2 ...
the version is that one. Shall I change packages.yaml?

Reply:
`3.8.2` everywhere, then I think this is good for merging